tools/tasks/mediaTasks.js

```diff
@@ -5,6 +5,7 @@ module.exports = function(gulp, config) {
     const path = require('path');
     const jsonmin = require('gulp-jsonmin');
     const merge = require('merge-stream');
+    const changelog = require('./copyChangelog');
```

Comment:
keep the naming consistent here.  if the file is called `copyChangelog` then the function should also be called `copyChangelog`.  Especially since that is the name of the exported function from that file